src/nupic/algorithms/Cells4.hpp

```diff
@@ -67,7 +67,6 @@ namespace nupic {
   namespace algorithms {
     namespace Cells4 {
 
-      class Cell;
```

Comment:
removed this dummy class, as it appeared nowhere! C++ tests pass OK w/o it. Also modified the py_Algorithms bindings accordingly